app/views/projects/show.html.erb

```diff
@@ -172,10 +172,9 @@
                 <% end %>
                 <hr class="projects-collaborators-horizontal-rule">
               </div>
+            <% end %>
           </div>
         </div>
-      </div>
-      <% end %>
```

Comment:
You deleted a `</div>` accidently. Restore that.

Reply:
Also, can you remove this `<div>` as well. Seems unnecessary
![image](https://user-images.githubusercontent.com/47032027/100547310-1ee3fc00-328c-11eb-86e1-9e6aac6b3d1d.png)